src/routes/tools/slr-coastal-inundation/_Map/Rasters.svelte

```diff
@@ -48,12 +49,11 @@
   });
 
   function addRasterLayers() {
-    rasterLayersProps.forEach(({ id, tileUrls, visibility }) => {
+    rasterLayersProps.forEach(({ tileUrls, visibility, slugs }) => {
       tileUrls.forEach((url, index) => {
         layerHandler.addMapLayer({
-          id,
+          id: slugs[index],
```

Comment:
using a combination of a layer id and and index value to create unique MapBoxGLJS layer and source ids seems error prone; it's better to just use a unique id from the start, hence using the tile layer slug instead.